modules/cluster-autoscaler-cr.adoc

```diff
@@ -66,8 +66,29 @@ If you do not specify a value, the default value of `1` is used.
 <14> Optional: Specify the period to wait before deleting a node after a node has recently been _deleted_. If you do not specify a value, the default value of `0s` is used.
 <15> Optional: Specify the period to wait before deleting a node after a scale down failure occurred. If you do not specify a value, the default value of `3m` is used.
 <16> Optional: Specify a period of time before an unnecessary node is eligible for deletion. If you do not specify a value, the default value of `10m` is used.
-<17> Optional:  Specify the _node utilization level_. Nodes below this utilization level are eligible for deletion. If you do not specify a value, the default value of `10m` is used.. The node utilization level is the sum of the requested resources divided by the allocated resources for the node, and must be a value greater than `"0"` but less than `"1"`. If you do not specify a value, the cluster autoscaler uses a default value of `"0.5"`, which corresponds to 50% utilization. This value must be expressed as a string.
-// Might be able to add a formula to show this visually, but need to look into asciidoc math formatting and what our tooling supports.
+<17> Optional:  Specify the _node utilization level_. Nodes below this utilization level are eligible for deletion. If you do not specify a value, the default value of `10m` is used.
++
+The node utilization level is the sum of the requested resources divided by the allocated resources for the node, and must be a value greater than `"0"` but less than `"1"`. If you do not specify a value, the cluster autoscaler uses a default value of `"0.5"`, which corresponds to 50% utilization. This value must be expressed as a string.
+<18> Optional: Specify any expanders that you want the cluster autoscaler to use.
+The following values are valid:
++
+--
+* `LeastWaste`: Selects the machine set that minimizes idle CPU after scaling.
+If multiple machine sets would yield the same amount of idle CPU, the selection minimizes unused memory.
+* `Priority`: Selects the machine set with the highest user-assigned priority.
+To use this expander, you must create a config map that defines the priority of your machine sets.
+For more information, see "Configuring a priority expander for the cluster autoscaler."
+* `Random`: (Default) Selects the machine set randomly.
+--
++
+If you do not specify a value, the default value of `Random` is used.
++
+You can specify multiple expanders by using the `[LeastWaste, Priority]` format.
+The cluster autoscaler applies each expander according to the specified order.
++
+In the `[LeastWaste, Priority]` example, the cluster autoscaler first evaluates according to the `LeastWaste` criteria.
+If more than one machine set satisfies the `LeastWaste` criteria equally well, the cluster autoscaler then evaluates according to the `Priority` criteria.
```

Comment:
I think "equally well" is not required. It does not add any value to the statement?

Reply:
Yeah, in this case it matters. To say just that it "satisfies" the criteria is a bit meaningless.  The lowest number will always "satisfy" it. What we care about here is when there is a tie for that lowest number - i.e., when more than one machine set satisfies the criteria equally.